hockeysdk/src/main/res/values-de/strings.xml

```diff
@@ -54,7 +54,7 @@
     <string name="hockeyapp_feedback_validate_email_empty">Bitte geben Sie eine E-Mail-Adresse an</string>
     <string name="hockeyapp_feedback_validate_name_error">Bitte geben Sie Ihren Namen an</string>
     <string name="hockeyapp_feedback_validate_text_error">Die Nachricht konnte nicht übertragen werden. Überprüfen Sie bitte Ihre E-Mail-Adresse.</string>
-    <string name="hockeyapp_feedback_generic_error">Es ist ein Fehler aufgetreten</string>
+    <string name="hockeyapp_feedback_generic_error" tools:ignore="UnusedResources">Es ist ein Fehler aufgetreten</string>
```

Comment:
It looks like `hockeyapp_feedback_generic_error` is not used in the SDK anymore. Can you please verify and remove the string if it's never used? (same is true for the other two strings.